scripts/generate_curriculum_from_images.py

```diff
@@ -0,0 +1,55 @@
+import argparse
+from pathlib import Path
+
+from PIL.Image import Resampling
+from yaml import load, dump
+
+IMAGE_SIZE = (640, 360)
+try:
+    from yaml import CLoader as Loader, CDumper as Dumper
+except ImportError:
+    from yaml import Loader, Dumper
+from PIL import Image
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser(
+        "Create new curriculum from directory full of objects. Take language "
+        "descriptions from filenames. "
+    )
+    parser.add_argument("images_dir", type=Path)
+    parser.add_argument(
+        "--output_dir", type=Path, default="data/multi_obj")
```

Comment:
This is fine for testing, but note that when you create the actual curricula for your experiment they must go under `data/curriculum/train` and `data/curriculum/test` respectively.